example/hooks.js

```diff
@@ -2,12 +2,12 @@ const aws = require('aws-sdk')
 const codedeploy = new aws.CodeDeploy({ apiVersion: '2014-10-06' })
 
 module.exports.pre = (event, context, callback) => {
-  var deploymentId = event.DeploymentId
-  var lifecycleEventHookExecutionId = event.LifecycleEventHookExecutionId
+  const deploymentId = event.DeploymentId
```

Comment:
ran `standard --fix` due to lint issues

![image](https://user-images.githubusercontent.com/18017094/131330599-41af499e-3197-45bc-af3c-36ed50198af7.png)